verifact/forum/schema.py

```diff
@@ -118,7 +126,53 @@ def mutate_and_get_payload(
 
         return AnswerCreate(answer=answer)
 
+class VoteUpdate(ClientIDMutation):
```

Comment:
Because VoteUpdate handles all CRUD operations, I wonder if a better name might not be something more inclusive, e.g. VoteCreateUpdateDelete (as long as it is). I think it would also be a good sanity check to add 3 tests for each of the cases (create, update, delete). The tests should be really quick to write now that we have a basic test suite in place. These tests also get run for every PR, so they will protect the repo in the future.

Reply:
yep will try to get that done tonight